staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go

```diff
@@ -171,10 +174,38 @@ func (t *prefixTransformers) TransformToStorage(ctx context.Context, data []byte
 	result, err := transformer.Transformer.TransformToStorage(ctx, data, dataCtx)
 	RecordTransformation("to_storage", string(transformer.Prefix), time.Since(start), err)
 	if err != nil {
+		logTransformErr(err, "failed to encrypt data", ctx)
 		return nil, err
 	}
 	prefixedData := make([]byte, len(transformer.Prefix), len(result)+len(transformer.Prefix))
 	copy(prefixedData, transformer.Prefix)
 	prefixedData = append(prefixedData, result...)
 	return prefixedData, nil
 }
+
+func logTransformErr(err error, message string, ctx context.Context) {
+	requestInfo := getRequestInfoFromContext(ctx)
+	if klogLevel6 := klog.V(6); klogLevel6.Enabled() {
+		klogLevel6.InfoSDepth(
```

Comment:
We're only logging when the err is not nil, so why is this `InfoSDepth` instead of `ErrorS...`?

Reply:
We need to skip the helper function to appropriately set the line number and is no ErrorSDepth